pyaedt/circuit.py

```diff
@@ -1402,15 +1411,15 @@ def load_diff_pairs_from_file(self, filename):
             return False
         return True
 
-    @pyaedt_function_handler()
-    def save_diff_pairs_to_file(self, filename):
+    @pyaedt_function_handler(filename="output_file")
+    def save_diff_pairs_to_file(self, output_file):
         """Save differtential pairs definition to file.
 
         If the file that is specified already exists, it is overwritten.
 
         Parameters
         ----------
-        filename : str
+        output_file : str
             Full qualified name of the file containing the differential pairs definition.
```

Comment:
```suggestion
            Full qualified name of the file to save the differential pairs definition to.
```
I'd recommend moving the following line to the parameter description:
"If the file that is specified already exists, it is overwritten."